cosmic-settings/src/pages/accessibility/mod.rs

```diff
@@ -170,18 +203,74 @@ pub fn vision() -> section::Section<crate::pages::Message> {
                     )
                     .map(crate::pages::Message::Accessibility),
                 )
+                .add(
+                    cosmic::Element::from(
+                        settings::item::builder(&descriptions[invert_colors]).control(
+                            toggler(page.screen_inverted).on_toggle_maybe(
+                                page.wayland_available
+                                    .is_some_and(|ver| ver >= 2)
+                                    .then_some(Message::SetScreenInverted),
+                            ),
+                        ),
+                    )
+                    .map(crate::pages::Message::Accessibility),
+                )
+                .add({
+                    cosmic::Element::from(
+                        settings::item::builder(&descriptions[color_filters]).control(
+                            toggler(page.screen_filter_active).on_toggle_maybe(
+                                page.wayland_available
+                                    .is_some_and(|ver| ver >= 2)
+                                    .then_some(Message::SetScreenFilterActive),
+                            ),
+                        ),
+                    )
+                    .map(crate::pages::Message::Accessibility)
+                })
+                .add({
+                    let selections = if page.screen_filter_selection == ColorFilter::Unknown {
+                        &page.screen_filter_selections
+                    } else {
+                        &page.screen_filter_selections[0..4]
+                    };
+                    cosmic::Element::from(
+                        settings::item::builder(&descriptions[color_filter_type]).control(
+                            dropdown(
```

Comment:
Should the dropdown be insensitive if the color filter toggle is off, or is intended to be able to select a filter, then enable it?

It should be insensitive when the protocol isn't present, anyway, like the toggles.

Reply:
I didn't see a way in the api to make it insensitive. Though I think it is nice from an UX perspective to be able to set and enable it in any order.

Reply:
Hm, yeah. Looking at the implementing of `Dropdown` in libcosmic I don't see any way to make it insensitive. Definitely something we'll want in libcosmic, but not a blocker here.